source/fundamentals/enterprise-auth.txt

```diff
@@ -194,7 +194,7 @@ to improve performance.
       :tabid: MongoCredential
 
       To cache Kerberos tickets by process, you must specify the ``JAVA_SUBJECT_PROVIDER``
-      mechanism property and provide a 
+      mechanism property and provide a
       `KerberosSubjectProvider <https://mongodb.github.io/mongo-java-driver/4.2//apidocs/mongodb-driver-core/com/mongodb/KerberosSubjectProvider.html#%3Cinit%3E()>`__
```

Comment:
Question: Any idea why this link seems to return a 404? (It works, but my link checker does not like it).
Question: Should this use the java driver directive and/or a source constant?

Suggestion: The link seems to contain a double slash:

```suggestion
      `KerberosSubjectProvider <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-core/com/mongodb/KerberosSubjectProvider.html#%3Cinit%3E()>`__
```

Reply:
I'll remove the slash, but the rest are out of scope of this ticket (fixing links).